go/saltpack/decrypt.go

```diff
@@ -18,8 +18,8 @@ type decryptStream struct {
 	mps        *msgpackStream
 	err        error
 	state      readState
-	keys       *receiverKeysPlaintext
-	sessionKey SymmetricKey
+	payloadKey SymmetricKey
```

Comment:
`payloadKey` is just a renaming of `sessionKey` to match the spec. The type hasn't changed. If you prefer the term "session key" though I can totally keep it.